src/openforms/js/components/admin/form_design/registrations/objectsapi/GenericObjectsApiVariableConfigurationEditor.js

```diff
@@ -0,0 +1,160 @@
+import {FieldArray, useFormikContext} from 'formik';
+import isEqual from 'lodash/isEqual';
+import PropTypes from 'prop-types';
+import React, {useContext} from 'react';
+import {FormattedMessage} from 'react-intl';
+import {useAsync, useToggle} from 'react-use';
+
+import {APIContext} from 'components/admin/form_design/Context';
+import {REGISTRATION_OBJECTS_TARGET_PATHS} from 'components/admin/form_design/constants';
+import Field from 'components/admin/forms/Field';
+import FormRow from 'components/admin/forms/FormRow';
+import Select, {LOADING_OPTION} from 'components/admin/forms/Select';
+import ErrorMessage from 'components/errors/ErrorMessage';
+import {post} from 'utils/fetch';
+
+import {asJsonSchema} from './utils';
+
+export const GenericEditor = ({
+  variable,
+  components,
+  namePrefix,
+  isGeometry,
+  index,
+  mappedVariable,
+  objecttype,
+  objectsApiGroup,
+  objecttypeVersion,
+  TargetPathDisplay,
+}) => {
+  const {csrftoken} = useContext(APIContext);
+  const [jsonSchemaVisible, toggleJsonSchemaVisible] = useToggle(false);
+
+  const {
+    loading,
+    value: targetPaths,
```

Comment:
Providing `targetPaths` with a default value allows it to be used, even when its still loading (this way, things like `getTargetPath` cannot break on the `.find(...)` function)
```suggestion
    value: targetPaths = [],
```